tests/parameter_handler/parameter_handler_6_bool.cc

```diff
@@ -41,7 +41,7 @@ int main ()
                          "docs 1");
       prm.leave_subsection ();
 
-      prm.read_input("prm");
+      prm.read_input(SOURCE_DIR "/prm/parameter_handler_6_bool.prm");
```

Comment:
It looks like it was right back before the ctest conversion (i.e., before bd1b0e5606): back then all the test files were called `prm` and had unique directories.